src/chocolatey.tests.integration/Scenario.cs

```diff
@@ -209,6 +209,11 @@ public static void create_directory(string directoryPath)
             _fileSystem.create_directory(directoryPath);
         }
 
+        public static void move_directory(string path, string newPath)
+        {
+            _fileSystem.move_directory(path, newPath);
+        }

```

Comment:
What was the reason for adding this? It doesn't seem to get called from anywhere.

Reply:
Was added as I was trying to make some tests to verify the "legacy" behaviour would still work, no longer needed if we're not keeping that.